src/SDKs/Relay/Management.Relay/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Relay Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Relay management functions for managing the Microsoft Azure Relay service")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.0.1.0")]
```

Comment:
@v-Ajnava usually you always bump up file version and only bump up AssemblyVersion when you bump up major version.